fabtests/multinode/src/core.c

```diff
@@ -55,9 +55,13 @@
 #include <assert.h>
 #include <hmem.h>
 
+#define BARRIER_SIZE	16
+#define BARRIER_KEY	0x1DEA
```

Comment:
Barrier doesn't need a message size.  Even MPI uses a 0-byte barrier.  We should just use a specific key for barrier messages to match them correctly.  I'd rename KEY to TAG and use this as a tag message.  A MR key isn't useful

Reply:
updated